ompi/mpiext/ftmpi/c/help-mpi-ft.txt

```diff
@@ -17,26 +18,6 @@ operation, yet you have requested support for fault tolerance.
 When using this component, normal failure free operation is expected;
 However, failures may cause the application to abort, crash or deadlock.
 
-In this framework, the following components are tested to operate under
```

Comment:
We would like to keep these. It is good that right now we do not have any components that fall into these 2 categories, but this does not means we should remove the text.

Reply:
✅   I'll put these messages back.